packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -688,3 +688,48 @@ export function isParentOf(element, targetElement) {
     false
   );
 }
+
+export async function copyToClipboard(text) {
+  if (navigator?.clipboard?.writeText) {
+    try {
+      await navigator.clipboard.writeText(text);
+    } catch (err) {
+      copyToClipboardFallback(text);
+    }
+  } else {
+    copyToClipboardFallback(text);
+  }
+}
+
+/**
+ * @param {string} text
```

Comment:
Why is this the only one with a typedef? Not that I think they should all have one either it just seems odd

Reply:
I'll fix that yeah all the functions need some comments and some typedefs👌